horreum-web/src/domain/tests/TestSettings.tsx

```diff
@@ -92,25 +101,25 @@ export default function TestSettings({ test, onTestIdChange, onModified, funcsRe
                 notificationsEnabled,
                 fingerprintLabels: [],
                 fingerprintFilter: undefined,
```

Comment:
With the modified UI,  changes to managing access are now sent sent via POST to `/api/test`, with the full test data.

These defaults are dropping `fingerprintLabels` and `fingerprintFilter` from the existing test, causing fingerprints to be re-calculated for the test

Reply:
Yeah because with the change, managing access would be like managing any other Test setting.

Do you see a problem in recalculating fingerprints even if we are just changing access rights? I mean I am not expecting many access rights changes on a test so I don't think that could be an issue, wdyt?

Reply:
Yes, if you select a test that has change detection variables set with a fingerprint label and change Access, the fingerprint labels are dropped and all the fingerprints are recalculated

Reply:
but isn't that the actual behavior if you change the Test setting?

Reply:
because if that was an issue also before (changing the settings will trigger the fingerprint recalculation, I still don't get if it was intended or not) I can easily fix that with something like:
```js
                fingerprintLabels: test?.fingerprintLabels || [],
                fingerprintFilter: test?.fingerprintFilter,
```

Reply:
Looking back, yes it is an existing bug, but I think we should fix it in this PR

Reply:
> but I think we should fix it in this PR

I agree, fixed exacty as I was suggesting above https://github.com/Hyperfoil/Horreum/pull/1720/files#diff-0596d0bd925fab40460a7fef8c88687a33fe10df0e2bf55ab77f92fab27eb9a5R102-R103